open-metadata-implementation/access-services/asset-lineage/asset-lineage-server/src/main/java/org/odpi/openmetadata/accessservices/assetlineage/handlers/ProcessContextHandler.java

```diff
@@ -180,68 +184,92 @@ public ProcessContextHandler(String serviceName,
                                                                                                                   PropertyServerException,
                                                                                                                   UserNotAuthorizedException {
-        List<EntityDetail> entities = new ArrayList<>();
+        boolean relationshipsExist = false;
         if (checkIfEntityExistWithSpecificType(entityDetails,PORT_ALIAS)) {
-            entities = endRelationship(entityDetails,userId);
+            relationshipsExist = endRelationship(entityDetails,userId);
         }
 
         if (checkIfEntityExistWithSpecificType(entityDetails,PORT_IMPLEMENTATION)) {
-            entities = getTabularSchemaTypes(entityDetails,userId);
+            relationshipsExist = getTabularSchemaTypes(entityDetails,userId);
         }
 
-        return entities;
+        return relationshipsExist;
     }
 
-
-    private List<EntityDetail> endRelationship(List<EntityDetail> entityDetails, String userId) throws InvalidParameterException,
+    /**
+     * Retrieves the entities that are connected to entityDetails that are passed.
+     * @param entityDetails      list of entities
+     * @param userId             String - userId of user making request.
+     * @return boolean true if relationships exist otherwise false.
+     */
+    private boolean endRelationship(List<EntityDetail> entityDetails, String userId) throws InvalidParameterException,
                                                                                                        PropertyServerException,
                                                                                                        UserNotAuthorizedException {
         List<EntityDetail> result = new ArrayList<>();
         for (EntityDetail entityDetail : entityDetails) {
-            result = getRelationshipsBetweenEntities(userId, entityDetail.getGUID(),
+            result.addAll(getRelationshipsBetweenEntities(userId, entityDetail.getGUID(),
                                            processRelationshipsTypes.get(entityDetail.getType().getTypeDefName()),
-                                           entityDetail.getType().getTypeDefName());
+                                           entityDetail.getType().getTypeDefName()));
         }
-        return result;
+        return result.isEmpty()? false: true;
     }
-    
-    private List<EntityDetail> getTabularSchemaTypes(List<EntityDetail> entityDetails, String userId) throws InvalidParameterException,
+
+    /**
+     * Retrieves the TabularSchemaTypes that are related to a Port Implementation Entity.
+     * @param entityDetails      list of entities
+     * @param userId             String - userId of user making request.
+     * @return boolean true if relationships exist otherwise false.
+     */
+    private boolean getTabularSchemaTypes(List<EntityDetail> entityDetails, String userId) throws InvalidParameterException,
```

Comment:
Same comment for all the get.. methods that return a boolean. They should be renamed to follow the naming conventions. Also, the javadoc might be misleading